### ISSUE TITLE
Pop up for download of a specific region

### ISSUE THREAD

**Issue by sandy99405** [organicmaps/organicmaps#6188]
### Describe the issue:
The app is asking for an extra download when zooming in for a specific region. This is consuming both time and space for the users.

### Steps to reproduce:
1. Preinstall all the regional maps before hand so that the user doesn't have to download all the maps individually after installing the app.
2. Reduce the size of the regional maps so that it won't increase the overall storage  space of the app.

### Screenshot:
Here it is asking to download the map of Berlin which is 74MB  in size.
[screenshot of the issue](https://ibb.co/j6CWrHp)

I am using the Android version of this app.

Can anybody please look into this issue ?

**Comment by biodranik** [comment]
1. Reducing maps size is a huge task. It's discussed in other issues. It requires downloading isolines (terrain) and wiki articles separately.
2. Not everyone wants to download maps automatically.
3. Regions should be downloaded automatically if you are connected using Wifi and zooming in to them.

**Comment by sandy99405** [comment]
@auge8472 and @biodranik you are absolutely right. Only because I use mobile data and have some storage issues, I thought it would be a good issue to raise. Also I wanted to work on some international issues like :
   [Passing loop at Cessnock #6129](https://github.com/organicmaps/organicmaps/issues/6129) and I had to download the Glasgow map.

 Although there were few issues on my side, I agree with your points. Closing this issue 👍.
